Makefile

```diff
@@ -80,61 +83,47 @@ GOTEST_INTEGRATION = $(GOTEST) -tags=integration
 
 # Integration test for sdk client
 .PHONY: test-sdk
-test-sdk: clean-sdk build-sdk
+test-sdk: clean-all build-all
 	DUMMY_BIN=$(PWD)/$(DUMMY_BIN) \
-	LOOKOUT_BIN=$(PWD)/$(LOOKOUT_BIN) \
+	LOOKOUT_BIN=$(PWD)/$(LOOKOUT_SDK_BIN) \
 	$(GOTEST_INTEGRATION) github.com/src-d/lookout/cmd/sdk-test
 
 # Same as test-sdk, but skipping tests that require a bblfshd server
 .PHONY: test-sdk-short
-test-sdk-short: clean-sdk build-sdk
+test-sdk-short: clean-all build-all
 	DUMMY_BIN=$(PWD)/$(DUMMY_BIN) \
-	LOOKOUT_BIN=$(PWD)/$(LOOKOUT_BIN) \
+	LOOKOUT_BIN=$(PWD)/$(LOOKOUT_SDK_BIN) \
 	$(GOTEST_INTEGRATION) -test.short github.com/src-d/lookout/cmd/sdk-test
 
 # Integration test for lookout serve
 .PHONY: test-json
-test-json: clean-sdk build-sdk
+test-json: clean build-all
 	DUMMY_BIN=$(PWD)/$(DUMMY_BIN) \
 	LOOKOUT_BIN=$(PWD)/$(LOOKOUT_BIN) \
 	$(GOTEST_INTEGRATION) github.com/src-d/lookout/cmd/server-test
 
 # Build sdk client and dummy analyzer
-.PHONY: build-sdk
-build-sdk: $(DUMMY_BIN) $(LOOKOUT_BIN)
+.PHONY: build-all
+build-all: $(DUMMY_BIN) $(LOOKOUT_BIN) $(LOOKOUT_SDK_BIN)
```

Comment:
maybe here we could call `.build`, `make -f Makefile.dummy build`, and `make -f Makefile.sdk build`

Reply:
that is the thing: calling `build` here would brake i.e server tests non-linux OSes - default binaries in `./build/bin` are ALWAYS for linux arch, so it will fail to exec with wired error.

Same for dummy. That another reason why only packaging is moved to separate makefile for dummy and sdk binaries, but this is kept for tests.